test/util/SE3_test.py

```diff
@@ -110,20 +110,18 @@ def test_is_approx(self):
         self.assertTrue(r3.rotation.is_approx(r7.rotation))
         self.assertFalse(r3.is_approx(r7))
         self.assertFalse(r7.is_approx(r3))
-    
+
     def test_transform_matrix(self):
         # test that an SE3 from a transform matrix is the same as an SE3
         # from an equivalent quaternion and position
-        r = np.array([[0, -1, 0],
-                      [1, 0, 0],
-                      [0, 0, 1]])
+        r = np.array([[0, -1, 0], [1, 0, 0], [0, 0, 1]])
         t = np.array([1, 2, 3])
         m = np.eye(4)
         m[:3, :3] = r
         m[:3, 3] = t
         s1 = SE3.from_transform_matrix(m)
 
-        q = np.array([0, 0, np.sqrt(2)/2, np.sqrt(2)/2])
+        q = np.array([0, 0, np.sqrt(2) / 2, np.sqrt(2) / 2])
```

Comment:
I know it's just formatting but I don't see why this should be here

Reply:
yea um, i just ran the style check oops